assets/css/pages.css

```diff
@@ -32,14 +32,30 @@
 
 .sponsor {
   column-span: 1;
-  padding-bottom: 85px;
+  padding-bottom: 50px;
   display: flex;
   justify-content: center;
   align-items: center;
   width: 33%;
   min-width: 250px;
 }
 
+.sponsor-tier-banner {
+  height: 15vh;
+  background: #2b2046ff;
+  width: 100%;
+  margin-bottom: 10vh;
```

Comment:
Also if this could be changed to 5vh -- it helps visually combine the header and logos.